docs/paper/dev/api/commands.mdx

```diff
@@ -0,0 +1,106 @@
+---
+slug: /dev/commands
+description: A guide to Paper's Brigadier command API.
+---
+
+# Command API
+
+Paper's command system is built on top of Minecraft's Brigadier command system. This system is a
```

Comment:
maybe link to some already existing (?) brigadier docs here so we dont need to explain stuff like how to use arguments and stuff